e2e/pages/organizations/onc-acbs-atls/organization.po.js

```diff
@@ -0,0 +1,86 @@
+const elements = {
+  organizationEditButton: '#edit-organization',
+  organizationName: '#organization-name',
+  organizationWebsite: '#organization-website',
+  saveOrganizationButton: '#chpl-organization-save',
+  organizationList: '.organizations-side-nav',
+  retireOrganization: '#organization-retired',
+  retirementDate: '#retirement-date',
+  addressOnEdit: 'chpl-address',
+  manageUsersPanelHeader: '//*[contains(text(),"Manage Users")]',
+  errorMessage: '.text-danger.ng-scope',
+};
+
+class OrganizationPage {
+  constructor() { }
+
+  organizationNameButton(organizationName) {
+    return $(`//*[contains(text(),"${organizationName}")]`);
+  }
+
+  get organizationList() {
+    return $(elements.organizationList);
+  }
+
+  organizationListCount() {
+    return $(elements.organizationList).length;
+  }
+
+  get organizationEditButton() {
+    return $(elements.organizationEditButton);
+  }
+
+  createOrganizationButton(organization) {
+    return $(`//*[contains(text(),"Create New ONC-${organization}")]`);
+  }
+
+  get organizationName() {
+    return $(elements.organizationName);
+  }
+
+  get organizationWebsite() {
+    return $(elements.organizationWebsite);
+  }
+
+  get saveOrganizationButton() {
+    return $(elements.saveOrganizationButton);
+  }
+
+  get retireOrganizationCheckbox() {
+    return $(elements.retireOrganization);
+  }
+
+  get retirementDate() {
+    return $(elements.retirementDate);
+  }
+
+  get manageUsersPanelHeader() {
+    return $(elements.manageUsersPanelHeader);
+  }
+
+  get manageUsersPanel() {
+    return $('chpl-users');
+  }
+
+  get newOrganizationGeneralInfo() {
+    return $('chpl-onc-organization');
+  }
+
+  retiredStatus(organizationType, organizationId) {
+    return $(`//*[@id="chpl-organization-ONC-${organizationType}-${organizationId}"]/ui-view/div/div`);
```

Comment:
I find locators that aren't xpath easier to understand. I don't think it's required to change it, but something like this should work too, and I think is easier to read:

```suggestion
    return $(`#chpl-organization-ONC-${organizationType}-${organizationId}/ui-view/div/div`);
```